lib/ui/pages/news_list/news_list_page.dart

```diff
@@ -27,46 +27,60 @@ class _NewsPageState extends State<NewsListPage> {
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            PrimaryNewsListItem(
-              news: PrimaryNewsListEntity(
-                imageUrl: 'https://i.picsum.photos/id/9/800/800.jpg',
-                title: "New laptops seem to be stuck to the board",
-                authorName: "Marek Aureliusz",
-                authorImageUrl:
-                    'https://www.ludoviccareme.com/files/image_211_image_fr.jpg',
-              ),
+            Container(
+              width: double.maxFinite,
+              height: 80,
+            ),
+            _buildNewsSectionHeader(
+              headerTitle: Strings.mostViewed,
+              routeId: null,
+            ),
+            SizedBox(height: _Constants.sectionHeaderPadding),
+            PrimaryNewsListView(
+              primaryNewsListEntities: primaryNewsListEntities.take(5).toList(),
+            ),
+            _buildNewsSectionHeader(
+              headerTitle: Strings.newest,
```

Comment:
taka drobnostka, ale w sumie jak metoda nazywa się _buildNewsSectionHeader to jakby parametr był po prostu `title` to z kontekstu można by było wyciągnąć, że chodzi o title headera, dlatego imo samo title wystarczy